12/umbraco-cms/content-delivery-api.md

```diff
@@ -0,0 +1,458 @@
+```
+
+#### Response
+
+{% code title="Expanded output for "linkedItem" property" %}
+```json
+{
+    "name": "My post",
+    "route": {
+        "path": "/my-post/",
+        "startItem": {
+            "id": "5d5ae914-9885-4ee0-a14b-0ab57f501a55",
+            "path": "demo-blog"
+        }
+    },
+    "id": "9bdac0e9-66d8-4bfd-bba1-e954ed9c780d",
+    "contentType": "blogpost1",
+    "properties": {
+        "title": "My post page",
+        "blogPostNumber": 11,
+        "bodyContent": "Congue, sollicitudin? Est fames maiores, sociis suspendisse et aliquet tristique excepturi, aliquam, nihil illum pretium penatibus exercitationem lacinia! Dolorem tempus convallis, nulla! Eius scelerisque voluptatum penatibus, dignissimos molestiae, soluta eum. Voluptatibus quod? Temporibus potenti voluptates dictumst? Cillum metus, nec asperiores? Impedit sit! Eum tellus cillum facilisis ullamco tempor? Sint nostrum luctus? Neque dictumst diam, minus? Itaque, minus, etiam dignissimos debitis occaecat aptent tempus! Praesent molestiae duis nihil recusandae, eius imperdiet aspernatur natus. Tempus mattis at architecto, augue, consequuntur ultricies eligendi, litora morbi ante nesciunt pretium laoreet quidem recusandae voluptates dapibus, iure sagittis donec ipsum mollit? Blanditiis! Laborum sit assumenda beatae.",
+        "linkedItem": {
+            "name": "Demo blog",
+            "route": {
+                "path": "/demo-blog/",
+                "startItem": {
+                    "id": "5d5ae914-9885-4ee0-a14b-0ab57f501a55",
+                    "path": "demo-blog"
+                }
+            },
+            "id": "5d5ae914-9885-4ee0-a14b-0ab57f501a55",
+            "contentType": "blog",
+            "properties": {
+                "title": "My demo blog",
+                "description": "Nihil incididunt dolores adipisicing placeat quisque imperdiet interdum autem, dolorem fusce rhoncus sunt leo inventore dictumst quisque, voluptatem, magni justo nostrud deserunt! Natus ipsam commodi dignissimos, sodales ab.\n"
+            }
+        }
+    },
+    "cultures": {}
+}
+```
+{% endcode %}
+
+
+
+The built-in property editors in Umbraco that allow for output expansion are:  &#x20;
+
+* `Umbraco.ContentPicker`
+* `Umbraco.MultiNodeTreePicker`
+
+</details>
+
+<details>
+
+<summary>Preview</summary>
+
+Similar to the preview concept in Umbraco, the Delivery API allows for requesting unpublished content through its endpoints. This can be done by setting a `Preview` header to `true` in the API request. However, accessing draft versions of your content nodes requires authorization via an API key configured in `appsettings.json` file - `Umbraco:CMS:DeliveryApi:ApiKey` setting. To obtain preview data, you must add the `Api-Key` request header containing the configured API key to the appropriate endpoints. Draft content is not going to be included in the JSON response otherwise.
+
```

Comment:
Would be nice with a request sample here that includes both the `Preview` and the `Api-Key` headers. Something like the request sample in the start item section.